### DISCUSSION TITLE
Documentation for Window control in 11.x docs

### DISCUSSION THREAD

**Discussion by MiKom** [AvaloniaUI/Avalonia#13354] [Q&A] [answered]
Hello.

I'm going through the tutorials for Avalonia 11 and it seems that the documentation for the `Window` control is missing.

It's available for 10.x: https://docs.avaloniaui.net/docs/controls/window
But it's gone in 11.x: https://docs.avaloniaui.net/docs/next/reference/controls/ (no `Window` in the menu on the left).

Is my assumption correct that 11.x docs are still a bit "WIP'y"? That's fine of course as 11.x is a recent release, just wanted to make sure that I'm looking at the right places.

Thanks,

**Comment by stevemonaco** [comment, marked as answer]
It seems like the `Window` docs were never remade for Avalonia 11. I can't find anything in the repo. You might want to post an issue over at [avalonia-docs](https://github.com/AvaloniaUI/avalonia-docs). And yes, the 11 docs are still quite WIP-y in a lot of areas.

**Reply by MiKom** [reply]
Thanks for confirmation. I filed an issue: https://github.com/AvaloniaUI/avalonia-docs/issues/209
